src/components/TaskNavBar/TaskNavBar.jsx

```diff
@@ -13,8 +21,14 @@ function TaskNavBar({ onCreateButtonClick }) {
         </button>
 
         <div className="task-navbar__filters">
-          {FILTER_TASKS_BUTTON_TEXT_ARRAY.map((filterButtonText, index) => (
-            <FilterButton buttonText={filterButtonText} key={index} />
+          {FILTER_TASKS_BUTTON_LABELS.map((filterButtonText, index) => (
+            <FilterButton
+              buttonText={filterButtonText}
+              key={index}
+              onClick={() =>
```

Comment:
Can we use some key other than index?

Reply:
@rsa-wtag vai, as there are only three items in the array, introducing another item as key is kind of unnecessary here, I think.